_includes/event-post-macros.html

```diff
@@ -0,0 +1,245 @@
+
+
+{# ==========================================================================
+
+   calendar_icon()
+
+   ==========================================================================
+
+   Description:
+
+   Create an HTML calendar icon when given:
+
+   date:           A valid date.
+
+   additional_classes: Extra classes you wish to add to the icon, space
+                       separated.
+
+   ========================================================================== #}
+
+{% macro calendar_icon(calendar_date, additional_classes) %}
+    <time class="calendar-icon {{ additional_classes }}" data-month="{{ calendar_date | date('%b') }}" data-day="{{ calendar_date | date('%d') }}" datetime="{{ calendar_date }}">
+        <span class="u-visually-hidden">{{ calendar_date | date('%b %d, %Y') }}</span>
+    </time>
+{% endmacro %}
+
+
+
+{# ==========================================================================
+
+   event_location_image()
+
+   ==========================================================================
+
+   Description:
+
+   Create HTML image when given:
+
+   post: A post from a query result.
+
+   options (optional): An object to use for customizing a event location image using
+                       Google Maps.
+
+   options.class:    A class name to be applied to the img element.
+
+   options.location: A location as either a comma-separated {latitude,longitude} 
+                     pair (e.g. "40.714728,-73.998672") or a string address 
+                     (e.g. "city hall, new york, ny"). 
+
+   options.scale:    Affects the number of pixels that are returned.    
+                     Accepted values are 1, 2 and 4.    
+
+   options.size:     Defines the rectangular dimensions of the map image. 
+                     {horizontal_value}x{vertical_value} (e.g. 500x400)
+
+   options.zoom:     Determines the magnification level of the map. 
+
+      
+
+   ========================================================================== #}
+
+{% macro event_location_image(post, options) %}
+
+    {%- set location = (post.location or 'Washington DC') -%}
+    {%- set scale = ('&scale=' + (options.scale or '1')) -%}
```

Comment:
This should probably default to 2 if we want these to be good on high-dpi displays.